jobs/JGDAS_ATMOS_VERFOZN

```diff
@@ -49,20 +39,27 @@ export TANKverf_ozn=${TANKverf_ozn:-${COM_ATMOS_OZNMON}}
 
 if [[ ! -d ${TANKverf_ozn} ]]; then mkdir -p -m 775 ${TANKverf_ozn} ; fi
 
+#-----------------------------------
+# Other variables
+#
+export OZN_AREA=${OZN_AREA:-glb}
+export oznstat=${oznstat:-${COM_ATMOS_ANALYSIS}/gdas.t${cyc}z.oznstat}
+export SATYPE_FILE=${SATYPE_FILE:-${FIXgdas_ozn}/gdas_oznmon_satype.txt}
+export PDATE=${PDY}${cyc}
+export DO_DATA_RPT=${DO_DATA_RPT:-1}
```

Comment:
Except `PDATE`, these should probably be moved to a config file.

Reply:
Moving.